### PR TITLE
Frontend dev

### PR DESCRIPTION
**Pull request by Thek10patil**
Tabs code added

### CODE REVIEW

**Comment by AdityaGovardhan** [comment]
Can you please resolve the conflicts and let me know if the conda_start.sh file is required?

File: src/tasks/task1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from django.views.generic import CreateView
+from src import models
+
+
+class Task1(CreateView):
+    model = models.Task1Model
+    fields = ('number_of_latent_semantics',)
+    template_name = 'task1.html'
+
+    def get_context_data(self, **kwargs):
```
Comment (vibnimit) [inline src/tasks/task1.py:11]: Can you explain what is the purpose of get_context_data and what is context here? Are you going to use it for active page in the front-end?

File: src/urls.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from django.urls import path
+
+from . import tasks
+
+urlpatterns = [
+    path('', tasks.index, name='index'),
+
+
+    path('task1', tasks.Task1.as_view(), name='task1'),
```
Comment (vibnimit) [inline src/urls.py]: what is the need for exposing two different URLs? We can have one url and then call the function inside and return the result, isn't it?

**Review by vibnimit — CHANGES_REQUESTED** [review]

File: src/tasks/task1.py (hunk 1 above)
Comment (Thek10patil) [reply src/tasks/task1.py:11]: To highlight active tab

**Comment by Thek10patil** [comment]
> Can you please resolve the conflicts and let me know if the conda_start.sh file is required?

conflict resolved

File: src/urls.py (hunk 2 above)
Comment (Thek10patil) [reply src/urls.py]: which 2 urls?

**Review by AdityaGovardhan — DISMISSED** [review]
looks clean to merge, although I have added one commit to remove conda_start.sh

File: src/tasks/task1.py (hunk 1 above)
Comment (vibnimit) [reply src/tasks/task1.py:11]: okay gotcha

File: src/urls.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    path('execute_task1', tasks.execute_task1, name='ExecuteTask1'),
+
+    path('task2', tasks.Task2.as_view(), name='task2'),
+    path('execute_task1', tasks.execute_task1, name='ExecuteTask2'),
```
Comment (vibnimit) [inline src/urls.py]: I think here it should be execute_task2. isn't it?

File: src/urls.py (hunk 2 above)
Comment (vibnimit) [reply src/urls.py]: path('task1', tasks.Task1.as_view(), name='task1'),
path('execute_task1', tasks.execute_task1, name='ExecuteTask1')

I think I got it 1st one is for displaying the form when Task1 is selected and the second one is for POST requests when the task is executed.

**Review by vibnimit — CHANGES_REQUESTED** [review]
Please checks the src/urls.py file there is typo I believe, if it is correct then update it otherwise we can go ahead

File: src/urls.py (hunk 3 above)
Comment (Thek10patil) [reply src/urls.py]: Done

**Review by vibnimit — APPROVED** [review]

**Review by vibnimit — APPROVED** [review]

**Review by AdityaGovardhan — APPROVED** [review]

**Pull request merged**